deps/3rdparty/libdispatch/src/queue.c

```diff
@@ -682,6 +717,13 @@ _dispatch_queue_dispose(dispatch_queue_t dq)
 	}
 #endif
 
+	if (slowpath(dq->dq_specific_q)) {
+		_dispatch_queue_dispose(dq_specific_q);
```

Comment:
I can't figure out how this compiles. `dq_specific_q` is a member of `dq`, but you are lacking a `->` here